assets/visualizing-with-bokeh/visualizing-with-bokeh.ipynb

```diff
@@ -0,0 +1,487 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Note: All output_file() calls have been replaced with output_notebook() so that plots will display inline.*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Basics of Bokeh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Your First Plot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from bokeh.plotting import figure, output_notebook, show\n",
+    "from bokeh.models.tools import LassoSelectTool, CrosshairTool, HoverTool\n",
+    "output_notebook()\n",
+    "\n",
+    "x = [1, 3, 5, 7]\n",
+    "y = [2, 4, 6, 8]\n",
+    "\n",
+    "p = figure()\n",
+    "\n",
+    "p.circle(x, y, size=10, color='red',  legend='circle')\n",
+    "p.line(x, y, color='blue', legend='line')\n",
+    "p.triangle(y, x, color='yellow', size=10, legend='triangle')\n",
```

Comment:
The yellow here is a bit difficult to read. Could you use an RGB hex value instead so as to specify a slightly darker shade of yellow? That would hopefully still work for accessibility but also increase the yellow/white contrast a little bit.

Reply:
@archaeocharlie Does this suggested adjust work for you?

Reply:
Would another of these named colors in the yellow family work? Perhaps gold or khaki?
https://bokeh.pydata.org/en/latest/docs/reference/colors.html#bokeh.colors.groups.yellow

Reply:
Those both look better to me. What about  you @drjwbaker?

Reply:
Gold? Who is making the change in the lesson file and ipynb?